scylladb/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/gofiber/storage/scylladb
+
+go 1.20
+
+require (
+	github.com/gocql/gocql v1.6.0
+	github.com/gofiber/utils v1.1.0
```

Comment:
Running `go mod tidy` should get rid of `gofiber/utils` 🤔 i dont see it being usef anywhere, right?